src/main/java/com/team/planopedia/API/translators/YelpAPI.java

```diff
@@ -196,5 +196,93 @@
 
         return tempReviewsData;
     }
+    
+        /**
+     * Method that makes a API call to the Yelp API to search for a specific restaurant given
+     * the name, city, and the number of results wanted
+     *
+     * @param _restaurantName breakfast, lunch, or dinner, etc.
```

Comment:
After trying to call this function, it seems like you can't have a space in the restrauntName string. Might be good to make note of that here